jekyll/_cci2/notifications.md

```diff
@@ -149,4 +149,8 @@ workflows:
       - build
 ```
 
+## Notification with RGB hardware
```

Comment:
```suggestion
## Third Party Notification Tools
```

Reply:
Renamed it to Third Party Tools with subhead Chroma Feedback